### PR TITLE
Migrate dashboards to Grafana Cloud

### PR DESCRIPTION
**Pull request by avnes**
- Add Cluster Health Dashboard
- Add cluster capacity
- Add more dashboards

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -9,14 +9,14 @@ locals {
 
 module "ce_folder" {
   #checkov:skip=CKV_TF_1:We rely on release tags
-  source = "git::https://github.com/dfds/terraform-grafana-cloud.git//grafana_folder?ref=0.3.2"
+  source = "git::https://github.com/dfds/terraform-grafana-cloud.git//grafana_folder?ref=0.4.0"
```
Comment (samidbb) [inline main.tf:12]: need to bump to latest version

File: main.tf [hunk 2]
```diff
@@ -9,14 +9,14 @@ locals {
...
   #source = "../../../../../../terraform-grafana-cloud//grafana_folder" # Support for local development
   title = var.folder_title
 }
 
 module "dashboards" {
   #checkov:skip=CKV_TF_1:We rely on release tags
-  source = "git::https://github.com/dfds/terraform-grafana-cloud.git//grafana_dashboard?ref=0.3.2"
+  source = "git::https://github.com/dfds/terraform-grafana-cloud.git//grafana_dashboard?ref=0.4.0"
```
Comment (samidbb) [inline main.tf:19]: need to bump to latest version

**Comment by samidbb** [comment]
need also to get latest from main

File: main.tf (hunk 1 above)
Comment (avnes) [reply main.tf]: Done

**Review by samidbb — APPROVED** [review]
lgtm

**Pull request merged**